backoffice/src/views/accountHolder/AccountHolderView.tsx

```diff
@@ -92,32 +91,44 @@ export const AccountHolderView: FunctionComponent<IAccountHolderViewProps> = ({
         const beacons = await accountHolderGateway.getBeaconsForAccountHolderId(
           id
         );
-        setAccountHolder(accountHolder);
-        setBeacons(beacons);
 
-        setLoading(false);
+        if (isMounted) {
```

Comment:
wouldn't this if statement always get execute as isMounted is always set to true at the top of useEffect?

Reply:
Good question! 

Shall we tuple?